File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] newArray = new int[2 * nums.length];

        for(int i = 0; i < nums.length * 2; i++) {
            if(i < nums.length) {
                newArray[i] = nums[i];
            } else {
                newArray[i] = nums[i - nums.length];
            }

        }
        return newArray; 
        
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        values = set(nums)
        max_set = 0

        for n in values:

            if((n - 1) not in values):
                current_count = 0
                current_value = n
                while(current_value in values):
                    current_count += 1
                    max_set = max(current_count, max_set)
                    current_value += 1
            else:
                continue
        return max_set 

                    

        
        